src/init.c

```diff
@@ -37,6 +39,24 @@ extern void _PG_fini(void);
 void
 _PG_init(void)
 {
+	if (!process_shared_preload_libraries_in_progress)
+	{
+	
+		char	   *force_load = GetConfigOptionByName("timescaledb.force_load", NULL, true);
```

Comment:
Actually this flag should be the opposite (if it's on then you can create extension without preload). The default should be we disallow installation without preload. maybe timescaledb.create_without_preload?